isaric/parsers/ccp-senegal.toml

```diff
@@ -0,0 +1,739 @@
+#:schema ../../schemas/dev/parser.schema.json
+
+[adtl]
+  name = "ccp-senegal"
+  description = "CCP Senegal"
+  defaultDateFormat = "%d/%m/%Y"
+
+  [adtl.tables.subject]
+    kind = "groupBy"
+    groupBy = "subject_id"
+    aggregation = "lastNotNull"
+    schema = "../../schemas/dev/subject.schema.json"
+    optional-fields = ["ethnicity"]
+
+  [adtl.tables.visit]
+    kind = "groupBy"
+    groupBy = "visit_id"
+    aggregation = "lastNotNull"
+    schema = "../../schemas/dev/visit.schema.json"
+
+  [adtl.tables.observation]
+    kind = "oneToMany"
+    schema = "../../schemas/dev/observation.schema.json"
+    common = { visit_id = { field = "record_id", sensitive = true } }
+
+[adtl.defs]
+  "Y/N/NK" = { values = { 1 = true, 2 = false } }
+  "né vivant/mort-né" = { values = { 1 = "né vivant", 2 = "mort-né" } }
```

Comment:
I don't think this is being used - but should make sure text is always returned in a single language (i.e. English for now).